src/taxonomy/taxonomy-card/TaxonomyCard.scss

```diff
@@ -30,6 +30,11 @@
     white-space: nowrap;
   }
 
+  .pgn__card-header-actions, .badge {
+    z-index: 100; 
+    position: relative;
+  }
+
```

Comment:
This change looks like it belongs on https://github.com/openedx/frontend-app-course-authoring/pull/645, not here?

Reply:
I needed this because the `stretched-link` was over the card menu and the tooltip from the system-defined badge. It was fixed using the clickable card (1499bb50f3c7eb964eb4f2752f05d856eced1865).